config/locales/en.yml

```diff
@@ -1,4 +1,11 @@
 en:
+  shared:
+    courses:
+      placements:
+        heading: School placements at %{provider_name}
+        warning: You can't pick which schools you will be in.
```

Comment:
Tiny tiny little thing -- in this codebase, we seem to use smart quotes. In the other codebase, we would say 'cannot'. Might be worth checking with Ruby what she would prefer.

Reply:
Actually, in the ticket, it uses smart quotes. Good spot, thank you